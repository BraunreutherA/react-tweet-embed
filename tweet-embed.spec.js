import test from 'ava'
import TweetEmbed from './tweet-embed'
import React from 'react'

test('renders', t => {
  t.snapshot(<TweetEmbed id='692527862369357824' />)
})

test.cb('calls twttr api', t => {
  global.window = {
    twttr: {
      widgets: {
        createTweetEmbed: (...args) => {
          t.snapshot(args)
          t.end()
        }
      }
    }
  }

  const comp = new TweetEmbed({id: 'tweet_id', options: {myOption: 1}})
  comp.componentDidMount()
})
