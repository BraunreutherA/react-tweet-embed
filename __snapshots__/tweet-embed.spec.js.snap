exports[`calls twttr api 1`] = `"["tweet_id",null,{"myOption":1}]"`;

exports[`renders 1`] = `"{\"key\":null,\"ref\":null,\"props\":{\"id\":\"692527862369357824\",\"protocol\":\"https:\",\"options\":{}},\"_owner\":null,\"_store\":{}}"`;
